testsuite/fhir.py

```diff
@@ -0,0 +1,33 @@
+""" FHIR Adapter """
+import requests
+
+
+OAUTH_URIS_DEFINITION = 'http://fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris'
+OAUTH_URIS_COM_DEFINITION = 'http://fhir-registry.smarthealthit.com/StructureDefinition/oauth-uris'
+
+
+def get_oauth_uris(base_url):
+    """ Conformance statement should define a set of oauth uris.
+    See: http://fhir-docs.smarthealthit.org/argonaut-dev/specification/#5
+    Params
+    ------
+    provider : researchapp.models.providers.Provider
+    Return
+    ------
+    dict :
+        authorize : string
+        token : string
+    """
+    url = '{url}metadata'.format(url=base_url)
+    headers = {
+        'Accept': 'application/json+fhir',
+    }
+    response = requests.get(url, headers=headers)
+    conformance = response.json()
```

Comment:
Can you try using https://github.com/smart-on-fhir/client-py (in pip: `fhirclient==v1.0.3`) to parse results, in general? (If it doesn't work, that would also be important to know.)